pennylane/devices/qubit/simulate.py

```diff
@@ -147,12 +158,20 @@ def get_final_state(circuit, debugger=None, **execution_kwargs):
     """
     prng_key = execution_kwargs.pop("prng_key", None)
     interface = execution_kwargs.get("interface", None)
+    initial_state = execution_kwargs.get("initial_state", None)
+
+    if initial_state is None:
+        circuit = circuit.map_to_standard_wires()
```

Comment:
Is this still needed if we've moved mapping to simulation indices higher in the stack?

Reply:
Not needed anymore.